examples/data/string-expression/string-expression.js

```diff
@@ -0,0 +1,46 @@
+const createSession = require('../../session');
+
+const qlikScript = `
```

Comment:
It's taken from other examples, let's rename this in all examples in a separate PR instead